readme.md

```diff
@@ -9,38 +9,75 @@ Features
 - Different output streams for .js, .d.ts files.
 - Support for sourcemaps using gulp-sourcemaps
 - Compile once, and filter different targets
-- Not just a wrapper around the ```tsc``` command, but a plugin that uses the TypeScript API.
 
 How to install
 --------------
+##### 1. Install Gulp
+```shell
+npm install --global gulp
+```
+##### 2. Install gulp in the project dependency
+```shell
+npm install gulp
+```
+##### 3. Install gulp-typeScript
 ```shell
 npm install gulp-typescript
 ```
 
-Easy usage
+Options
+-------
+- ```out``` (string) - Generate one javascript and one definition file. Only works when no module system is used.
+- ```outDir``` (string) - Move output to a different (virtual) directory. Note that you still need `gulp.dest` to write output to disk.
+- ```removeComments``` (boolean) - Do not emit comments to output.
+- ```noImplicitAny``` (boolean) - Warn on expressions and declarations with an implied 'any' type.
+- ```noLib``` (boolean) - Don't include the default lib (with definitions for - Array, Date etc)
+- ```noEmitOnError``` (boolean) - Do not emit outputs if any type checking errors were reported.
+- ```target``` (string) - Specify ECMAScript target version: 'ES3' (default), 'ES5' or 'ES6'.
+- ```module``` (string) - Specify module code generation: 'commonjs' or 'amd'.
+- ```sourceRoot``` (string) - Specifies the location where debugger should locate TypeScript files instead of source locations.
```

Comment:
I saw this option was still in the readme, but it should be removed as it has been removed recently. Can you remove it here?